### PR TITLE
[Logging]Init systemAccountManager for logging tester deployer

### PR DESCRIPTION
**Pull request by aiwantaozi**
Problem:
Input fluentd configure in advance mode got crash
Solution:
This bug was introduced when update systemAccountManager for system apps, need to init systemAccountManager for tester deployer

Issue:
https://github.com/rancher/rancher/issues/20568

### CODE REVIEW

**Comment by orangedeng** [comment]
LGTM

File: pkg/controllers/user/logging/deployer/testerdeployer.go [hunk 1]
```diff
@@ -48,7 +50,7 @@ func (d *TesterDeployer) Deploy(level, clusterName, projectID string, loggingTar
 	}
 
 	systemProjectID := ref.Ref(systemProject)
-	creator, err := d.systemAccountManager.GetProjectSystemUser(systemProject.Name)
+	creator, err := d.systemAccountManager.GetSystemUser(systemProject.Spec.ClusterName)
```
Comment (cjellick) [inline pkg/controllers/user/logging/deployer/testerdeployer.go]: can you explain why you need to switch to the cluster-wide user rahter than project user?

Comment (aiwantaozi) [reply pkg/controllers/user/logging/deployer/testerdeployer.go]: Hi, this app will deployed to system project, and the chart has clusterrole rbac, project user couldn't create clusterroles rbac.
https://github.com/rancher/system-charts/blob/dev/charts/rancher-logging/0.1.1/charts/fluentd-tester/templates/clusterrole.yaml

Comment (cjellick) [reply pkg/controllers/user/logging/deployer/testerdeployer.go]: That is a good explanatio. Can you add that to the commit message? You don't need to reference the chart via a github link, but just add something like:
```
This app will deployed to system project, and the chart has clusterrole rbac, project user couldn't create clusterroles rbac.
```

Comment (aiwantaozi) [reply pkg/controllers/user/logging/deployer/testerdeployer.go]: Updated. Thanks

**Comment by cjellick** [comment]
lgtm

**Pull request merged**